alertlogic-cli

```diff
@@ -9,12 +9,15 @@ import traceback
 
```

Comment:
ran `make lint` it didn't notice changes in alertlogic-cli. When i do `pycodestyle alertlogic-cli`:
```
pycodestyle alertlogic-cli
alertlogic-cli:7:15: E401 multiple imports on one line
alertlogic-cli:25:31: E401 multiple imports on one line
alertlogic-cli:33:36: E401 multiple imports on one line
```

Reply:
Fixed in setup.cfg